Util/LinkExtensions.cs

```diff
@@ -24,7 +24,8 @@ public static class LinkExtensions
         /// <returns></returns>
         public static MvcHtmlString ResourceLink(this HtmlHelper helper, string path)
         {
-            return new MvcHtmlString(String.Format("//{0}/{1}", ConfigurationManager.AppSettings["Domain"], path));
+            var port = ConfigurationManager.AppSettings["Port"];
+            return new MvcHtmlString($"//{ConfigurationManager.AppSettings["Domain"]}:{port}/{path}");
```

Comment:
Please only add the port if it's not 80 (if the current request is https) or 443 (if the current request is via https)